docs/manage-validator/maxHealthChecks.md

```diff
@@ -0,0 +1,180 @@
+---
+id: maxHealthChecks
+title: Enhance Validator Reliability maxHealthChecks
+sidebar_label: Validator reliability maxHealthChecks
+---
+
+In the ever-evolving landscape of Ethereum infrastructure, maintaining robust and reliable validator operations is paramount. Validators play a crucial role in securing the network, proposing blocks, and verifying the state of the chain. However, challenges like beacon node unavailability or connectivity issues can disrupt these operations, leading to potential downtime or unsafe behaviors. To address such concerns, we introduce a significant improvement for safer validator shutdowns and restarts, with a key highlight being the new `maxHealthChecks` configuration. These changes underwent rigorous testing using tools such as Kurtosis, simulating scenarios such as beacon node failures and recoveries to verify graceful handling without panics or indefinite hangs.
```

Comment:
this first paragraph reads a bit strange for an instructional page, we should have it read like instructions for a feature